climada/hazard/tc_tracks.py

```diff
@@ -666,10 +666,15 @@ def equal_timestep(self, time_step_h=1, land_params=False):
                                       itertools.repeat(land_geom, self.size),
                                       chunksize=chunksize)
         else:
-            new_data = list()
+            last_perc = 0
+            new_data = []
             for track in self.data:
-                new_data.append(self._one_interp_data(track, time_step_h,
-                                                      land_geom))
+                perc = 100 * len(new_data) / len(self.data)
+                if perc - last_perc >= 10:
+                    LOGGER.info("Progress: %d%%", perc)
+                    last_perc = perc
+                new_data.append(
+                    self._one_interp_data(track, time_step_h, land_geom))
```

Comment:
Not related to PR, but maybe use list comprehension here:

`new_data = [_one_interp_data(track, time_step_h, land_geom) for track in self.data]`

This would however remove the progress indicator. I am not 100% convinced by having by default so much logging, so I would be happy with the list comprehension.

Reply:
I think verbose output is helpful for debugging. What do you think about changing the logging level to `debug`?